library/xwiki

```diff
@@ -6,13 +6,13 @@ GitCommit: fbc0fe3cbc07119c45ae8ea97c7e021d41e89b0a
 Directory: 8/mysql-tomcat
 
 Tags: 9, 9.8, 9.8.1, 9-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable
-GitCommit: 088aaf39de290ece849eabace996b21f3b99d954
+GitCommit: ed8536e2861198c2052f54634ada97723239f5e6
 Directory: 9/mysql-tomcat
```

Comment:
This is not correct @evalica 

We only support LTS + the latest stable, so it should be:

```
Tags: 9, 9.9, 9-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable
GitCommit: ed8536e2861198c2052f54634ada97723239f5e6
```